docs/src/user-docs/guides-mcp-server/mcp-cursor.md

```
@@ -0,0 +1,29 @@
+# Connecting Cursor
+
+This guide explains how to connect [Cursor](https://cursor.com/) to
+the CLP MCP server over HTTP.
+
+## Steps
+
+1. Follow the [official Cursor guide](https://cursor.com/docs/context/mcp) 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)